tests/xapi_test_case.php

```diff
@@ -77,6 +77,7 @@ protected function get_transformer_config() {
             'send_mbox' => false,
             'send_response_choices' => false,
             'send_short_course_id' => false,
+            'send_course_and_module_idnumber' => false,
```

Comment:
@davidpesce this reminds me that we should probably have some tests that check these config values still work in the plugin when they're enabled.